docs/example.md

```diff
@@ -0,0 +1,108 @@
+# Full workflow tutorial
+This example deploys sample aplication on OpenShift,
+exports that aplication to Nulecule, and than deploys exported applicatoin to Kubernetes.
+
+This has been tested on ADB 1.7.0 as Fedora 23 on local machine
+
+## Deploy Docker registry on your local machine
+ 1. Run Docker registry on you local machine.
+    - `docker run -d -p 5000:5000 --name registry registry:2`
+
+
+## Start ADB and deploy sample OpenShift application
+ 1. Clone the ADB repository
+
+    `git clone https://github.com/projectatomic/adb-atomic-developer-bundle.git`
+
+ 1. Run OpenShift instance in ADB
+
+    - `cd adb-atomic-developer-bundle/components/centos/centos-openshift-setup`
+    - `vagrant up`
+    - `vagrant ssh`
+
+ 1. Add Docker Registry on your local machine as insecure
+     
```

Comment:
Its little unclear where to run this?

`If its to be run on my bare metal host:`
Please move this section up, put it before '*Start ADB and deploy sample OpenShift application*' and after '*Deploy Docker registry on your local machine*'

`else:`
Change the name **local machine** to **ADB box**.